### DISCUSSION TITLE
Some install.sh fixes

### DISCUSSION THREAD

**Discussion by Kurdakov** [ThioJoe/YT-Spammer-Purge#305] [General]
@lolpro11 
Currently I'm not on linux (at home and don't want to install terminal ubuntu),
but looks like install.sh and install_full.sh need tag updates

TAG = sudo -u $USER curl https://api.github.com/repos/ThioJoe/YT-Spammer-Purge/releases/latest -s | jq .name -r
and then
sudo -u $USER curl https://codeload.github.com/ThioJoe/YouTube-Spammer-Purge/tar.gz/refs/tags/v$TAG -o yt-spammer.tar.gz
...
sudo -u $USER cd YouTube-Spammer-Purge-$TAG/

the problem is jq should be installed before hand, so in all install commands like
apt install python3 python3-tk

jq must be added

apt install python3 python3-tk jq

**Comment by lolpro11** [comment]
[#326](https://github.com/ThioJoe/YT-Spammer-Purge/pull/326)

I added a check for version. Once done, it removes jq using `apt purge jq`.

Also, python check and a root check were added! :D
